README.md

```diff
@@ -194,6 +217,12 @@ Now Puppet knows where to download the module from when you include it in your s
     # github "java",     "1.6.0"
     include java
 
+### Updating modules
+
+* To check for modules updates defined in your `Puppetfile`, run `bundle exec librarian-puppet outdated`. You'll probably need a Github API Token [see above](#github-api-token) to avoid Github's API rate limits.
```

Comment:
Instead of suggesting users to deal with token by themself would be better to wrap a boxen command for this, since boxen already has an internal token?